mxcube3/routes/loginutils.py

```diff
@@ -73,7 +73,10 @@ def logged_in_users(exclude_inhouse=False):
     users = [user["loginID"] for user in mxcube.USERS.itervalues()]
 
     if exclude_inhouse:
-        ih_users =["%s%s"% (p, c) for (p, c) in mxcube.session.in_house_users]
+        if type(mxcube.session.in_house_users[0]) == tuple:
```

Comment:
Pay attention to that the session that we have today have been written to handle proposal code and number it would probably not scale to handle the kind of user accounts that you currently are using.

There is no issue with this snippet but keep it in mind, because it might cause you a problem later, for instance when generating paths ...

Reply:
I have not seen anything on this, probably because we inherited the session hwobj. But thanks for the hint anyway! You never know